service/service.go

```diff
@@ -306,7 +325,11 @@ func (app *Application) shutdownExtensions() {
 }
 
 func (app *Application) execute() {
-	app.logger.Info("Starting...", zap.Int("NumCPU", runtime.NumCPU()))
+	app.logger.Info("Starting "+app.info.LongName+"...",
```

Comment:
Instead of logging this information at start-up time, can we instead add a `-v` flag which displays it and exits?

Reply:
I think we can do it in addition to logging.

I find it problematic that logs don't contain this information. When I am troubleshooting a collector that is deployed and running by someone else and I only have access to the logs it is very difficult to do proper troubleshooting without knowing the version and build of the collector running.